api/rest/restapi.go

```diff
@@ -654,6 +666,33 @@ func (api *API) unpinHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (api *API) pinPathHandler(w http.ResponseWriter, r *http.Request) {
+	if ps := api.parsePathOrError(w, r); ps.Cid != "" {
+		logger.Debugf("rest api pinPathHandler: %s", ps.Cid)
+
+		err := api.rpcClient.Call("",
+			"Cluster",
+			"Pin",
```

Comment:
We need to provide `PinPath` and `UnpinPath` in the main Cluster component. So we can expose those via RPC and this should call `Cluster.PinPath` (similar to pinHandler).